### PR TITLE
problem with test report

### PR DESCRIPTION
**Pull request by pavel-gutsal**
https://pavel-gutsal.github.io/layout_stars/
https://pavel-gutsal.github.io/layout_stars/report/html_report/
unfortunatly pixel perfect test fails
me and Artem Topolski tried to resolve this error but we failed .
with his pemission I pushed it on github as it is.
thanks for understanding.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
```
Comment (ArturErmolenko) [inline src/style.css:13]: это дефолтное значение его можно не указывать

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+}
```
Comment (ArturErmolenko) [inline src/style.css:43]: селекторы с одинаковыми стилями можно записать через запятую 
```
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
   background-image: url("/images/star-active.svg");
 }
```

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
```
Comment (vkryvytskyy) [inline src/style.css:15]:
```suggestion
.stars__star {
```

**Review by vkryvytskyy — APPROVED** [review]

**Comment by pavel-gutsal** [comment]
mentioned issue is fixed

**Pull request closed without merging**